index.yaml

```diff
@@ -13233,6 +13233,134 @@ entries:
     - assets/rancher-logging-crd/rancher-logging-crd-3.6.000.tgz
     version: 3.6.000
   rancher-monitoring:
+  - annotations:
+      artifacthub.io/license: Apache-2.0
+      artifacthub.io/links: |
+        - name: Chart Source
+          url: https://github.com/prometheus-community/helm-charts
+        - name: Upstream Project
+          url: https://github.com/prometheus-operator/kube-prometheus
+      artifacthub.io/operator: "true"
+      catalog.cattle.io/auto-install: rancher-monitoring-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/deploys-on-os: windows
+      catalog.cattle.io/display-name: Monitoring
+      catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.28.0-0'
+      catalog.cattle.io/namespace: cattle-monitoring-system
+      catalog.cattle.io/permits-os: linux,windows
+      catalog.cattle.io/provides-gvr: monitoring.coreos.com.prometheus/v1
+      catalog.cattle.io/rancher-version: '>= 2.8.0-0 < 2.9.0-0'
+      catalog.cattle.io/release-name: rancher-monitoring
+      catalog.cattle.io/requests-cpu: 4500m
+      catalog.cattle.io/requests-memory: 4000Mi
+      catalog.cattle.io/type: cluster-tool
+      catalog.cattle.io/ui-component: monitoring
+      catalog.cattle.io/upstream-version: 45.31.1
+    apiVersion: v2
+    appVersion: v0.65.1
+    created: "2024-02-02T16:39:37.22521-03:00"
```

Comment:
I think the `created` field should also match with what is in `dev-v2.8`. When do you `git merge release-v2.8`, there will be merge conflicts in index.yaml. You copy the monitoring entry version `103.0.3+up45.31.1` part of the index from `release-v2.8` and paste it and then you copy the version `103.0.2+up45.31.1` in `dev-v2.8` as the second entry of `rancher-monitoring` in index.yaml. Then save the file and `git add`. This should solve the issue here. Let me know if it works.

Reply:
You're right, it makes sense. I have changed the timestamp to match the one in the release like you said